CHANGELOG.rst

```diff
@@ -2,6 +2,67 @@
 Changelog
 =========
 
+3.0.0-alpha.0 (2024-10-24)
+==========================
+
+This is an alpha release, meaning it is not finished yet or suitable for production use.
+
+Detailed changes
+----------------
+
+**New features**
+
+* [:backend:`4770`] Obtain submission price from variable. There is an extra option
+  for setting the price of a product, by using a form variable. The variable can only
+  be of numeric data type (integer, float) and can be chosen in the pricing logic of
+  the form.
```

Comment:
ah, I was confused - this ID is the PR and not the ticket. Let's drop this entry in favour of line 28. Ideally a changelog entry is just a single line! If people want to learn more, they are supposed to search in the rest of the documentation ;)